app/controllers/alchemy/admin/languages_controller.rb

```diff
@@ -1,11 +1,28 @@
 module Alchemy
   module Admin
     class LanguagesController < ResourcesController
+      def index
+        @query = Language.on_current_site.ransack(params[:q])
+        @languages = @query.result.page(params[:page] || 1).per(per_page_value_for_screen_size)
```

Comment:
As we use this pattern all over the place, I would like to address this in a separate PR